### ISSUE TITLE
Snatched

### ISSUE THREAD

**Issue by filoor** [CouchPotato/Homey-App#1]
Homey do's net see when a movie is snatched

**Comment by RuudBurger** [comment]
How did you set up your webhook? Still need to see how this can be done the best way.

**Comment by filoor** [comment]
return true;

**Comment by filoor** [comment]
Downloaded also don't work i tink you right somethings is wrong with webhook

**Comment by RuudBurger** [comment]
If you use the "test" button in CouchPotato settings, Homey should say something.
If that doesn't work, the other stuff also won't

**Comment by filoor** [comment]
but if a do search all movies in wanted list that works

**Comment by RuudBurger** [comment]
Yeah that is the other way Homey -> CouchPotato. The notifications are CouchPotato -> webhook -> Homey. Working on getting that easier installed, so I'll have a look at this issue also.

**Comment by filoor** [comment]
Thanx
